validator/json_parser.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+from validator.constants import FIREFOX_GUID
+
+
+class ManifestJsonParser():
```

Comment:
Also see https://docs.python.org/release/2.5.2/ref/node33.html for more information.